build.gradle

```diff
@@ -5,14 +5,9 @@ buildscript {
 
   dependencies {
     classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
I kept the basic `Android` plugin at the top level, because the sample apps in the repo use it.